megamek/src/megamek/common/weapons/LRMSmokeWarheadHandler.java

```diff
@@ -90,11 +90,26 @@ protected boolean specialResolution(Vector<Report> vPhaseReport,
         // Handle munitions.
         if (atype.getMunitionType() == AmmoType.M_SMOKE_WARHEAD) {
             int damage = wtype.getRackSize() * calcDamagePerHit();
+            int radius = (int) Math.round(damage / 10.0);
```

Comment:
Did you mean the smoke round of SRM/LRM/MML, not the artillery? On the errata all smoke rounds of SRM/LRM/MML only fill a single hex:

> - Smoke Missiles fill a single target hex with a column of smoke 2 levels high (see p. 47)
> -This smoke is treated as Light Smoke if the weapon delivers 5 points or less in damage to the target hex. Otherwise, it is treated as Heavy Smoke
> - This smoke dissipates in the End Phase of the third turn after the attack.

Reply:
Errata change over time. If you cite errata you need to give a version number. The latest is 3.01, released 6 June 2019, and does not contain that change. Either it was reverted or accidentally omitted, but this is the current rule.

Reply:
It is on 3.0 and also on 3.01 as well. Please check pg. 75-76 of [errata 3.01](https://www.dropbox.com/s/tje4tr6bhsb8kxv/Tactical%20Operations%20-%202019-06-06%20-%20v3.01.pdf?dl=0) Check 'Smoke Missiles (Standard LRMs/Standard SRMs/MMLs) (p.371)'

Reply:
So it is. I just double-checked it this morning and don't know how I missed it.

Reply:
Well, there are so many lines out there so it may happen.